numpy/core/src/multiarray/compiled_base.c

```diff
@@ -1823,17 +1855,22 @@ unpack_bits(PyObject *input, int axis, PyObject *count_obj)
         npy_intp index;
         unsigned const char *inptr = PyArray_ITER_DATA(it);
         char *outptr = PyArray_ITER_DATA(ot);
+        lookup *unpack_lookup;
+        unpack_lookup = &unpack_lookup_b;
```

Comment:
Put this in an `else` clause of the `if` statement below.

Reply:
fixed